tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -50,7 +51,6 @@ def with_python_base(context: PipelineContext, python_image_name: str = "python:
         context.dagger_client.container()
         .from_(python_image_name)
         .with_mounted_cache("/root/.cache/pip", pip_cache)
-        .with_mounted_directory("/tools", context.get_repo_dir("tools", include=["ci_credentials", "ci_common_utils"]))
```

Comment:
Python package local and transitive dependencies are now automatically resolved with `find_local_python_dependencies`

Reply:
Nice!